packages/cli/test/utils/simulation/assertions/withdrawalsAssertion.ts

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import {capella} from "@lodestar/types";
+import {ApiError} from "@lodestar/api";
+import {MAX_WITHDRAWALS_PER_PAYLOAD} from "@lodestar/params";
+import {AssertionMatch, AssertionResult, SimulationAssertion} from "../interfaces.js";
+
+type WithdrawalsData = {
+  withdrawalCount: number;
+  withdrawalAmount: bigint;
+  validators: Record<number, {balanceInLastSlot: bigint; currentBalance: bigint; withdrawalAmount: bigint}>;
+};
+
+export function createWithdrawalAssertions<T extends string>(
+  nodeId: T
+): SimulationAssertion<`withdrawals_${T}`, WithdrawalsData> {
+  return {
+    id: `withdrawals_${nodeId}`,
+    match({forkConfig, epoch, node}) {
+      if (nodeId === node.id && epoch === forkConfig.CAPELLA_FORK_EPOCH)
```

Comment:
I'm surprised this line pass lints (no curly brasses).

Reply:
It was actually written as single line, but prettier changed it to this format. And our ESlint don't complaint on Prettier formatting.